tests/conftest.py

```diff
@@ -74,58 +74,74 @@ def base_route(api_version, gordo_project, gordo_name):
     return f"/gordo/{api_version}/{gordo_project}/{gordo_name}"
 
 
+@pytest.fixture(scope="session")
+def model_collection_directory(gordo_project: str):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        collection_dir = os.path.join(tmp_dir, gordo_project)
+        os.makedirs(collection_dir, exist_ok=True)
+        yield collection_dir
```

Comment:
Would this maybe be a good opportunity to use pytest's built in tmpdir?

Reply:
I have an Issue I opened on going through and doing all tmpdir swaps in one fell swoop.

Reply:
ya, but that doesn't mean we have to repeat the error until _that_ issue is fixed, right?

Reply:
Sorry, looking at this in the code again... we shouldn't/can't use the pytest built-in, as it's function scoped, and this is sessions scope. That was why I session scoped the "custom" tmp_dir, figuring we could use that one when we wanted a session-scoped temp dir.

Reply:
It seems like my last comment didn't get posted here. 

Looking at this again, the reason I didn't do that is that the pytest fixture is function scoped, which was why I session-scoped our "custom" tmp_dir fixture, so we could use that when we needed a session scoped tmp_dir, such as here.